x/auth/keeper/grpc_query_test.go

```diff
@@ -191,3 +191,61 @@ func (suite *KeeperTestSuite) TestGRPCQueryParameters() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestGRPCQueryModuleAccounts() {
+	var (
+		req *types.QueryModuleAccountsRequest
+	)
+
+	testCases := []struct {
+		msg       string
+		malleate  func()
+		expPass   bool
+		posttests func(res *types.QueryModuleAccountsResponse)
+	}{
+		{
+			"success",
+			func() {
+				req = &types.QueryModuleAccountsRequest{}
+			},
+			true,
+			func(res *types.QueryModuleAccountsResponse) {
+				var stakeModuleExists = false
+				for _, acc := range res.Accounts {
+					var account types.AccountI
+					err := suite.app.InterfaceRegistry().UnpackAny(acc, &account)
+					suite.Require().NoError(err)
+
+					moduleAccount, ok := account.(types.ModuleAccountI)
+
+					suite.Require().True(ok)
+					if moduleAccount.GetName() == "mint" {
+						stakeModuleExists = true
```

Comment:
Minor adjustment. `stakeModuleExists` should be `mintModuleExists` based on the above statement.